test/linter/test-support-history.js

```diff
@@ -0,0 +1,71 @@
+import chalk from 'chalk';
```

Comment:
This might fix the problem:

```suggestion
import chalk from 'chalk-template';
```